content/doc/developer/plugin-governance/adopt-a-plugin.adoc

```diff
@@ -0,0 +1,98 @@
+  In such case we can transfer ownership for plugins being hosted within the `jenkinsci` GitHub organization.
+  Such transfer may take a while.
+
+== Requesting commit access
+
+Email the http://jenkins-ci.org/content/mailing-lists[Jenkins Developers mailing list] and request to be made a maintainer 
+(which usually means commit access to the plugin repository and being made default assignee for newly reported issues in JIRA).
+Please provide the following things in the request:
+
+* Link to a plugin you want to adopt
+* Link(s) to pull requests you want to deliver, if applicable
+* Your GitHub username/id (e.g. oleg-nenashev for https://github.com/oleg-nenashev/)
+* Your Jenkins infrastructure account id. link:https://accounts.jenkins.io/[Create your account] if you don't have one.
+
+For abandoned plugins, you are expected to try and reach out to existing maintainer(s) using a best effort.
+The typical way to do that is to ping in GitHub and to add her/his/their email addresses in CC (hint: Git commits should have this information).
+We typically wait for about 2 weeks in normal work periods before proceeding, so please be patient.
+Hence, if you can prove the existing maintainer already agrees and you explicitly asked about taking over (e.g. in a PR discussion), the process can be faster.
+
+Once granted access, you can file a pull request against link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permission updater] to be able to deploy snapshots and releases for your plugin.
+You're generally expected to start slowly, by filing PRs, and not commit directly.
+Even more for plugins with a big number of installations for obvious reasons.
+
+== FAQ
+
+=== A plugin I'm (planning on) using shows up as looking for a maintainer. Does that mean I shouldn't use it?
+
+*No.* Jenkins is designed with backward compatibility in mind, so it's rare that a plugin stops working.
+And even then there's often someone who can fix the bug and release a new version even if they wouldn't be considered a maintainer of the plugin.
+So if you're happy with what a plugin is offering, there's no reason not to use it just because it's up for adoption.
+
+=== I want to help! How can I find a plugin to maintain?
+
+Check out the list of plugins up for adoption at the bottom of this page.
+If you see a plugin you like, visit its wiki page as it may contain additional information about the adoption request.
+
+=== I know which plugin I want to help with, what should I do now?
+
+Once you've chosen a plugin, review the documentation on plugin maintainership in the Jenkins project. 
+This is especially important if you're not currently a plugin developer.
+
+As a new maintainer of a new plugin, you'll inherit its existing users, so be careful with breaking changes.
+https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-Compatibilitymatters[We
```

Comment:
https://jenkins.io/project/governance/#compatibility-matters

Reply:
Just make a suggestion so that I merge it from phone :)

Reply:
I was also on the phone and couldn't suggest 😉 
```suggestion
link:/project/governance/#compatibility-matters[We
```